src/pymmcore_widgets/_mda/_util/_labeled_slider.py

```diff
@@ -0,0 +1,127 @@
+from qtpy import QtCore, QtGui, QtWidgets
+
+
+class QLabeledSlider(QtWidgets.QWidget):
+    """Slider that shows name of the axis and current value."""
+
+    valueChanged = QtCore.Signal([int], [int, str])
+    sliderPressed = QtCore.Signal()
+    sliderMoved = QtCore.Signal()
+    sliderReleased = QtCore.Signal()
+
+    def __init__(
+        self, name: str = "", orientation=QtCore.Qt.Horizontal, *args, **kwargs
+    ):
+        # super().__init__(self, *args, **kwargs)
+        super().__init__()
+        self.name = name
+
+        self.label = QtWidgets.QLabel()
+        self.label.setText(name.upper())
+        self.label.setAlignment(QtCore.Qt.AlignRight)
+        self.label.setSizePolicy(
+            QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Fixed
+        )
+        self.slider = QtWidgets.QSlider(orientation)
+        for function in [
+            "blockSignals",
+            "setTickInterval",
+            "setTickPosition",
+            "tickInterval",
+            "tickPosition",
+            "minimum",
+            "maximum",
+            "setTracking",
+            "value",
+        ]:
+            func = getattr(self.slider, function)
+            setattr(self, function, func)
+
+        self.current_value = QtWidgets.QLabel()
+        self.current_value.setText("0")
+        self.current_value.setAlignment(QtCore.Qt.AlignLeft)
+        self.current_value.setSizePolicy(
+            QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Fixed
+        )
+
+        self.play_btn = QtWidgets.QPushButton("▶")
```

Comment:
ultimately, i'd like to use a font icon rather than a UTF8 glyph here.  see here for an example:

https://github.com/pymmcore-plus/pymmcore-widgets/blob/54c240c2e41f5fb27dfb0a085e22f2215def6d18/src/pymmcore_widgets/_snap_button_widget.py#L74

`MDI6.play` and `MDI6.stop` would be a good match I suppose